prusti-viper/src/encoder/builtin_encoder.rs

```diff
@@ -81,7 +80,8 @@ impl<'p, 'v: 'p, 'tcx: 'v> BuiltinEncoder<'p, 'v, 'tcx> {
         }
     }
 
-    pub fn encode_builtin_method_def(&self, method: BuiltinMethodKind) -> vir::BodylessMethod {
+    pub fn encode_builtin_method_def(&self, method: BuiltinMethodKind) -> EncodingResult<vir::BodylessMethod> {
```

Comment:
Why was this changed to a `Result` return? There are no error paths. Is this just for consistency?

Reply:
Because I have some `BuiltinMethodKind` that I want to add that need to return a `Result`.